ravenframework/Samplers/Sampler.py

```diff
@@ -204,6 +205,7 @@ def __init__(self):
     self.distDict                      = {}          # Contains the instance of the distribution to be used, it is created every time the sampler is initialized. keys are the variable names
     self.funcDict                      = {}          # Mapping between variable name and the a 2-element namedtuple namedtuple('func', ['methodName', 'instance']) containing:
                                                      # element 0 (methodName): name of the method in the function to be be invoked. Either the default "evaluate", or the function name
+    self.variableFunctionExecutionList = []          # This is an ordered sequence of functional variable (linked to functions) that need to be performed (in case of interdependency). This list is always created. If no interdependence is detected, the order is just random, otherwise the order is determined through graph theory. 
```

Comment:
```suggestion
    self.variableFunctionExecutionList = []          # This is an ordered sequence of functional variable 
                                                                              # (linked to functions) that need to be performed (in case of 
                                                                              # interdependency). This list is always created. If no interdependence
                                                                              # is detected, the order is just random, otherwise the order is 
                                                                              # determined through graph theory. 
```